rewrite-java/src/main/java/org/openrewrite/java/tree/Space.java

```diff
@@ -36,40 +33,25 @@
  */
 @EqualsAndHashCode
 @JsonIdentityInfo(generator = ObjectIdGenerators.IntSequenceGenerator.class, property = "@ref")
-public class Space implements Markable {
-    public static final Space EMPTY = new Space("", emptyList(), Markers.EMPTY);
-
-    /**
```

Comment:
Why did these flyweights go away?

Reply:
Previously the flyweight map was _only_ used for a `Space` with no comments, no whitespace, and non-empty markers.
A very narrow usage of the pattern. I'm not even sure if there are real world conditions where a `Space` matching those criteria exist. So I got rid of it. 

But this probably needs _more_ flyweight usage, not less. I'll see about putting in expanded flyweight usage.

Reply:
I put the flyweights back in. I shouldn't have removed them in the first place.